tests/system/python/api/test_common.py

```diff
@@ -61,6 +61,7 @@ def test_ping_default(self, reset_and_start_fledge, fledge_url):
         assert 'green' == jdoc['health']
         assert jdoc['authenticationOptional'] is True
         assert jdoc['safeMode'] is False
+        assert '1.8.2' == jdoc['version']
```

Comment:
instead of changing with every release, I would suggest unit tests to have [regex match check for semVer ](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string). We don't use prerelease/buildmetadata, hence we can skip those group matches even if optional.  A simple check that version exists is also okay.

Reply:
Agreed, a simple [0-9]+\.[0-9+]\.[0-9]+ regex check would be good enough.

Reply:
Unit tests are updated in https://github.com/fledge-iot/fledge/pull/243/commits/0733b9588ee62d3f626f1b1bc25f7281edab46c3, to have a check contains as those are ultimately mocked. For API integration test, we can add a patch PR for this regex check.

Reply:
Added under https://github.com/fledge-iot/fledge/pull/244